src/Features/Lsif/Generator/Program.cs

```diff
@@ -127,30 +136,37 @@ private static async Task LocateAndRegisterMSBuild(TextWriter logFile)
             MSBuildLocator.RegisterInstance(msbuildInstance);
         }
 
-        private static async Task GenerateFromProjectAsync(FileInfo projectFile, ILsifJsonWriter lsifWriter, TextWriter logFile)
+        private static async Task GenerateFromProjectAsync(
+            FileInfo projectFile, ILsifJsonWriter lsifWriter, TextWriter logFile, CancellationToken cancellationToken)
         {
             await GenerateWithMSBuildWorkspaceAsync(
                 projectFile, lsifWriter, logFile,
                 async w =>
                 {
                     var project = await w.OpenProjectAsync(projectFile.FullName);
```

Comment:
Should this be passing through cancellation tokens?

```suggestion
                async (w, cancellationToken) =>
                {
                    var project = await w.OpenProjectAsync(projectFile.FullName, cancellationToken);
```

Reply:
Indeed.  i'm not sure waht's up with the analyzer.  It doesn't seem to be reporting all cases where it could pass a CT along...